Legiones Astartes.cat

```diff
@@ -611,6 +611,74 @@ If at any point there are only Rapier Carrier Models remaining in this Unit, tho
                     </entryLink>
                   </entryLinks>
                 </selectionEntry>
+                <selectionEntry type="upgrade" import="true" name="Banestrike bolter w/. Bayonet" hidden="true" id="c882-902f-dd51-67f5" sortIndex="2">
```

Comment:
This needs a max1/min1 on it. 
<img width="421" height="138" alt="image" src="https://github.com/user-attachments/assets/c949e9a5-2615-4f7b-89ee-2cc21167899b" />